core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -131,7 +165,7 @@ export class QuestionBackendApiService {
   /**
    * Checks if given input is a list and has all strings
    */
-  private isListOfStrings(list): boolean {
+  private isListOfStrings(list: unknown): list is string[] {
```

Comment:
I don't see "unknown" type being used in other places. Can you please explain why we need it? Ditto below.

Reply:
done